ClockDisplay.java

```diff
@@ -79,6 +82,6 @@ public String getTime()
     private void updateDisplay()
     {
         displayString = hours.getDisplayValue() + ":" + 
-                        minutes.getDisplayValue();
+                        minutes.getDisplayValue() + " " + amPm;
```

Comment:
As long as there are no problems in setting the original string this should work,